roles/openshift_grafana/defaults/main.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # We should probably use something more official here.
-openshift_grafana_image: "docker.io/mrsiano/grafana-ocp:latest"
+openshift_grafana_image: "docker.io/mrsiano/openshift-grafana-centos:5.2.0"
```

Comment:
Is it normal to ship images under individual user container repos?

Reply:
currently yes, but we gonna change this very soon.